docker-compose.yml

```diff
@@ -40,24 +42,39 @@ services:
     networks:
       - app-network
   mongodb:
-    image: mongo
-    container_name: ee461l_mongoDB
+    image: mongo:latest
+    container_name: ee461l_mongodb
+    hostname: ee461l_mongodb
     restart: unless-stopped
     command: mongod --auth
     environment:
       MONGO_INITDB_ROOT_USERNAME: "admin"
       MONGO_INITDB_ROOT_PASSWORD: "password"
-      MONGO_INITDB_DATABASE: ee461l_DB
-      MONGODB_DATA_DIR: /data/db
+      MONGO_INITDB_DATABASE: ee461l_mongodb
+      MONGODB_DATA_DIR: /data/dir
     volumes:
       - mongodbdata:/data/dir
+    ports:
+      - "27017:27017"
+    networks:
+      - app-network
+  mongo-express:
+    image: mongo-express:latest
+    container_name: mongo_express
+    ports:
+      - "8081:8081"
+    environment:
+      ME_CONFIG_MONGODB_ADMINUSERNAME: admin
+      ME_CONFIG_MONGODB_ADMINPASSWORD: password
+      ME_CONFIG_MONGODB_SERVER: ee461l_mongodb
+    depends_on:
+      - mongodb
```

Comment:
This thing is awesome. Had no idea it existed.

Reply:
Found it while researching some of the issues I was having, its sweet. I may also look into setting up swagger to help with API documentation if you've used that before